test/helpers.go

```diff
@@ -18,7 +18,7 @@ func loadMockJsons(commaDelimitedFilenames, pathToJsons string) ([][]byte, error
 	jsonFilenames := strings.Split(commaDelimitedFilenames, ",")
 	var bytesArray [][]byte
 	for _, jsonFilename := range jsonFilenames {
-		fullPath := path.Join("./features/unit/", pathToJsons, jsonFilename)
```

Comment:
Instead of the other helper, I think this can be hard coded to `./features/resources`

Reply:
If they end up in features/resources.